controllers/shooter.py

```diff
@@ -88,9 +89,13 @@ def tracking(self) -> None:
             # self.next_state("searching")
             self.state = self.searching
         else:
-            if abs(vision_data.angle) > self.find_allowable_angle(vision_data.distance):
+            old_turret_angle = self.turret.azimuth_at_time(self.vision.timestamp)
+            current_turret_angle = self.turret.azimuth_at_time(time.monotonic())
```

Comment:
Maybe get this one first, as it's time sensitive, then get the old one, as it isn't. But there's not much in it.

Reply:
It feels like there should be a dedicated method for getting specifically the current azimuth as well.